src/main.ts

```diff
@@ -39,7 +39,21 @@ async function run(): Promise<void> {
       options
     );
 
+    let helmfileOutput = '';
+    let helmfileError = '';
```

Comment:
I would personally call these two as `helmfileStdout` and `helmfileStderr` respectively.